<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterBitNameDialog</class>
 <widget class="QDialog" name="RegisterBitNameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register BitName</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Reserve</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditReserveName">
         <property name="placeholderText">
          <string>Enter BitName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditReserveAddress">
         <property name="placeholderText">
          <string>Enter Address</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="AmountField" name="amountReserveFee"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonReserve">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Reserve</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Register</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEditRegisterName">
         <property name="placeholderText">
          <string>Enter BitName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCommitment">
         <property name="placeholderText">
          <string>Enter Commitment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditIPv4">
         <property name="placeholderText">
          <string>Enter IPv4 Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRegisterAddress">
         <property name="placeholderText">
          <string>Enter Address</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Fee:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="AmountField" name="amountRegisterFee"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRegister">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AmountField</class>
   <extends>QLineEdit</extends>
   <header location="global">qt/amountfield.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
